furusiya/model/keys/util.py

```diff
@@ -0,0 +1,20 @@
+from game import Game
+
+
+def in_game(callback=None, *, pass_turn=False):
+    """
+    Decorator for ensuring in-game access only
+    Also marks the current turn as passed.
+    """
+    def decorator(callback):
+        def _inner_function(event):
+            if Game.game_state == 'playing' and Game.current_turn is Game.player:
```

Comment:
Please write tests for this. Basically, assert that the `Game.current_turn` value becomes none (or not) if the state is playing etc.